Makefile

```diff
@@ -246,7 +247,7 @@ check: config/check.$(VERSION).sha patch $(CHECK_FILES)
             color = 196;   \
         system("tput setaf " color "; printf " $$2 "; tput sgr0"); \
         printf " ]\n"; \
-    }' | column --separator '\t' --table
+    }' | column --separator $$'\t' --table
```

Comment:
This is a minor fix to properly match tab characters when formatting the `check` output.